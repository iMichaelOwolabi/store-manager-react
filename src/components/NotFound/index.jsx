import React from 'react';
import { Link } from 'react-router-dom';
import './NotFound.scss';

export default function NotFound() {
  return (
    <div id="notfound">
      <div className="notfound">
        <div className="notfound-404">
          <h2>Page Not Found!</h2>
        </div>
        <Link to="/">
          <h2>Back to Homepage</h2>
        </Link>
      </div>
    </div>
  );
}
