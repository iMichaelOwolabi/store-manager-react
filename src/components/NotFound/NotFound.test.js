import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './index';

describe('Test the 404 Component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
