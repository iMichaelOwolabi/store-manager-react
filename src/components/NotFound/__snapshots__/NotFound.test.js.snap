// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the 404 Component should match snapshot 1`] = `
<div
  id="notfound"
>
  <div
    className="notfound"
  >
    <div
      className="notfound-404"
    >
      <h2>
        Page Not Found!
      </h2>
    </div>
    <Link
      replace={false}
      to="/"
    >
      <h2>
        Back to Homepage
      </h2>
    </Link>
  </div>
</div>
`;
